.editorconfig

```
@@ -0,0 +1,23 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
```

I believe this should generally be true, that all files should end on a newline? Inherited from c, also a convention to get reduced noise in git diffs, and I know better for interacting with files from shell too...
```suggestion
insert_final_newline = true
```